common/culture/cultures/trogg.txt

```diff
@@ -133,5 +133,56 @@
 			{ name = "mercenary_company_band_of_horned_welshmen" }
 		}
 	}
+
+	drogbar = {
+		graphical_cultures = {
+			tibetan_group_coa_gfx
+			indian_building_gfx
+			mongol_clothing_gfx
+			mongol_unit_gfx
+			
+			drogbar_portrait_gfx
+		}
+		color = rgb { 0 176 250 }
+
+		cadet_dynasty_names = {
+			"dynn_Elwynn"
+		}
+
+		dynasty_names = {
+			"dynn_Wrynn"
+		}
+
+		male_names = {
+			Amgrol Amprol Arbrogg Armazkul Armol Blelgol Blevofrun Blorgarbugg Bubgrol Clalmugg Clolufrogg Clomgrub Dlalurbul Dlizgrugg Ezarol Ezgrul Gelprogg Halgogg Hudrugg 
+			Inbrugg Jankrogg Jiruzkol Kagamprub Kerbul Koggrugg Komprolg Lemprogg Lilbrugg Lozgrugg Lumgrob Mamgrugg Mamrul Mardul Nozgozkugg Torurgrugg Ukrun Ulabgrob Urmogg 
+			Urogg Zirgogg
+		}
+		female_names = { }
```

Comment:
We still need to invent female names.

Reply:
In-game they don't have females. Should they in the mod?

Reply:
> In-game they don't have females. Should they in the mod?

They should because somehow they should reproduce.

Reply:
> > In-game they don't have females. Should they in the mod?
> 
> They should because somehow they should reproduce.

maybe its possible to have they be listed as something other then male/female? the mechanics can still work the same but just an idea

Reply:
> 
> 
> > > In-game they don't have females. Should they in the mod?
> > 
> > 
> > They should because somehow they should reproduce.
> 
> maybe its possible to have they be listed as something other then male/female? the mechanics can still work the same but just an idea

That's quite a good idea